src/test/java/Homework14.java

```diff
@@ -0,0 +1,46 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework14 extends BaseTest {
+
+    @Test
+    public static void Navigation() throws InterruptedException{
+        //test: registration page navigation
+        //precondition: chrome browser is opened
+        //step 1: navigate to koel login page
+        //step 2: click registration link
+        //expected result: user should be taken to koel registration page
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.